akka-projection-core/src/test/scala/akka/projection/TestTags.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
+
+import org.scalatest.Tag
+
+object TestTags {
+
+  object InMemoryDb extends Tag("InMemoryDb")
+  object ContainerDb extends Tag("ContainerDb")
+  object FlakyDb extends Tag("FlakyDb")
```

Comment:
For the moment I'm only excluding those marked with FlakyDb. We may consider running the ContainerDb only in cron job, for instance. 

Let's see how useful this will be for us.

Reply:
:+1: